code/modules/VR/vr_avatar.dm

```diff
@@ -0,0 +1,87 @@
+
+
+/mob/living/carbon/human/virtual_reality
+	var/mob/living/carbon/human/real_me //The human controlling us, can be any human (including virtual ones... inception...)
```

Comment:
I did not know that. Lol. Yea I was redoing it to work with dynamic space allocation for VR rooms.